app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Expr.scala

```diff
@@ -59,9 +60,12 @@ sealed trait Expr extends Product with Serializable {
   }
 
   def leafs[T <: PageMode](formModel: FormModel[T]): List[Expr] = this match {
-    case Add(field1: Expr, field2: Expr)           => field1.leafs(formModel) ++ field2.leafs(formModel)
-    case Multiply(field1: Expr, field2: Expr)      => field1.leafs(formModel) ++ field2.leafs(formModel)
-    case Subtraction(field1: Expr, field2: Expr)   => field1.leafs(formModel) ++ field2.leafs(formModel)
+    case Add(field1: Expr, field2: Expr)         => field1.leafs(formModel) ++ field2.leafs(formModel)
```

Comment:
Is this a formatting issue...=> doesn't seem to be aligned

Reply:
No, this is scalafmt formatted